src/strings/en-us.json

```diff
@@ -1170,6 +1173,7 @@
     "PasswordResetComplete": "The password has been reset.",
     "PasswordResetConfirmation": "Are you sure you wish to reset the password?",
     "PasswordResetHeader": "Reset Password",
+    "PasswordResetProviderHelp": "Choose a Password Reset Provider to be used if this user resets their password",
```

Comment:
what does this "if" mean? like "choose which provider to use when in a distant future this user wishes to change password", or "choose which provider to use right now to reset the password"?

Reply:
The former

Reply:
Then maybe put it like `Choose a Password Reset Provider to be used when this user requests a password reset`

Reply:
It would also be nice to manually trigger a password reset if you want to add that.

Reply:
Using @JustAMan's phrasing, Adding a button to trigger a password reset is a project for later, but definitely might be worth doing